src/docs/product/issues/issue-details/performance-issues/large-http-payload/index.mdx

```diff
@@ -19,6 +19,10 @@ The detector for this performance issue looks for a single http span that meets
 
 If Sentry isn't detecting a Large HTTP Payload issue where you expect one, it's probably because the transaction didn't meet one of the above criteria.
```

Comment:
```suggestion
If Sentry isn't detecting a large HTTP payload issue where you expect one, it's probably because the transaction didn't meet one of the above criteria.
```